src/Analysis/Codelyzer.Analysis.Build/WorkspaceBuilderHelper.cs

```diff
@@ -459,6 +461,25 @@ private EnvironmentOptions GetEnvironmentOptions(IProjectFile projectFile)
                 }
             }
 
+            try
+            {
+                MSBuildLocator.RegisterDefaults();
```

Comment:
this is Windows only API.... had sync with Mark, we are getting rid of buildlocator... so I am assuming we are not using this API anymore...

Reply:
Yes we are not going to use this anymore.